crowbar.yml

```diff
@@ -26,6 +26,13 @@ barclamp:
 crowbar:
   layout: 1
   order: 99
-  run_order: 99
-  chef_order: 99
+  run_order: 2000
+  chef_order: 2000
   proposal_schema_version: 3
+
+nav:
+  utils:
+    upgrade:
+      order: 50
+      route: 'upgrade_path'
```

Comment:
As this action updates the proposals this should be a post action with a confirmation.